Stratis.Bitcoin/BlockStore/BlockStoreFeature.cs

```diff
@@ -57,13 +58,16 @@ public static class MBlockStoreBuilderExtension
 	{
 		public static IFullNodeBuilder UseBlockStore(this IFullNodeBuilder fullNodeBuilder)
 		{
-			fullNodeBuilder.ConfigureFeature(features =>
+            var blockRepository = new BlockRepository(fullNodeBuilder.Network, new DataFolder(fullNodeBuilder.NodeSettings));
```

Comment:
I realize BlockRepository is already injected so really just call blockRepository.Initialize() from the feature strat